apps/bbsync/query.py

```diff
@@ -136,14 +135,6 @@ def generate_description(self):
             # TODO
             # self._query["comment_is_private"] = True if ... else False
 
-    def generate_resolution(self):
```

Comment:
This makes sense to be removed. You should however also remove the `status` from the query on the line 92